### DISCUSSION TITLE
Timeout errors with whisper-large and long processing times.

### DISCUSSION THREAD

**Discussion by nishanthcmesh** [deepgram/community#272] [General help] [answered]
While using whisper-cloud with the `whisper-large` model, I kept getting timeout errors for a ~1 hr audio file. 
I even tried breaking the file into chunks of 2.5 minutes and the errors continued, for ~7 chunks out of 23 that I had. 
In addition, latency was very high, taking 48 minutes for a 57 minute file, with some segments timing out. 
The chunks were `audio/mp4` files of 2.5 minutes each for context.

1. Is there some kind of fix for these timeouts and high latencies?

2. How do I set the timeout parameter using the deepgram python sdk?

3. Is there some kind of rate limit/higher latency for free tier accounts? We are considering using Deepgram in our app and it is unusable with these many errors and timeouts. I checked the status page for both times when I tried and DG whisper cloud was listed as operational. 

4. Whisper-large transcription quality was way better for us compared to Nova. We were trying with Indian English speakers, is Nova not optimized for Indian accents?

**Comment by uanandaraja** [comment]
The latency and timeout error for Whisper has been occurring for me too for the last 4 days, at least. Hopefully, this get resolved soon. Maybe no one is resolving it due to weekend?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @nishanthcmesh and @uanandaraja, intermittent higher Whisper latency is a known occurrence as we manage our overall traffic. Your approach to chunk files will help, as will converting any video files to audio-only (if applicable). Nova will have much lower latency than Whisper.

Nova supports several dialects of English. For Indian English speakers, using `en-IN` rather than `en` or `en-US` will increase accuracy. See our full [Languages](https://developers.deepgram.com/docs/languages-overview) docs.
